core/translations/order.json

```diff
@@ -228,7 +228,6 @@
   "226": "device_name__change_template",
   "227": "device_name__title",
   "228": "entropy__send",
-  "229": "entropy__title",
```

Comment:
Keep the key in `order.json` so that the indices in translation blobs are not mangled. In more detail here: https://github.com/trezor/trezor-firmware/pull/4118#discussion_r1727108772